### ISSUE TITLE
ReturnTypes\ModelExtension::getTypeFromStaticMethodCall() returns a boolean

### ISSUE THREAD

**Issue by szepeviktor** [nunomaduro/larastan#50]
```
   Symfony\Component\Debug\Exception\FatalThrowableError  : Return value of NunoMaduro\Larastan\ReturnTypes\ModelExtension::getTypeFromStaticMethodCall() must implement interface PHPStan\Type\Type, boolean returned

  at /home/viktor/src/gallerytool/vendor/nunomaduro/larastan/src/ReturnTypes/ModelExtension.php:115
    111|                 }
    112|             }
    113|         }
    114|
  > 115|         return $returnType;
    116|     }
    117|
    118|     /**
    119|      * Replaces Static Types by the provided Static Type.

  Exception trace:

  1   NunoMaduro\Larastan\ReturnTypes\ModelExtension::getTypeFromStaticMethodCall(Object(Mockery_0_PHPStan_Reflection_Php_PhpMethodReflection_PHPStan_Reflection_Php_PhpMethodReflection), Object(PhpParser\Node\Expr\StaticCall), Object(PHPStan\Analyser\Scope))
      /home/viktor/src/gallerytool/vendor/phpstan/phpstan/src/Analyser/Scope.php:1253

  2   PHPStan\Analyser\Scope::resolveType(Object(PhpParser\Node\Expr\StaticCall))
      /home/viktor/src/gallerytool/vendor/phpstan/phpstan/src/Analyser/Scope.php:352
```

After debugging it turned out in [line 110](https://github.com/nunomaduro/larastan/blob/master/src/ReturnTypes/ModelExtension.php#L110) `$types` is **empty** array.

**Comment by szepeviktor** [comment]
First `$types` is

```
array (
  0 =>
  PHPStan\Type\ObjectType::__set_state(array(
     'className' => 'Illuminate\\Database\\Eloquent\\Model',
  )),
)
```
then `replaceStaticType()` makes it an empty array.

**Comment by nunomaduro** [comment]
@szepeviktor Do you know the exactly src code/or the exact file in your project that fires that issue?

**Comment by szepeviktor** [comment]
Yes, it is a private project, in your Gmail inbox now.

**Comment by szepeviktor** [comment]
@nunomaduro Maybe Gmail does not deliver .php files.

**Comment by nunomaduro** [comment]
@szepeviktor can you sent it like raw text?

**Comment by nunomaduro** [comment]
@szepeviktor Thanks again for reporting this false positive. 👍

This issue **got fixed** on the version `v0.1.4`. Do you mind of performing a `composer update nunomaduro/larastan` and test it again?

**Comment by szepeviktor** [comment]
Thank you, it is solved.
